pandas/_libs/window_indexer.pyx

```diff
@@ -0,0 +1,165 @@
+# cython: boundscheck=False, wraparound=False, cdivision=True
+
+import numpy as np
+from numpy cimport ndarray, int64_t
+
```

Comment:
is there anything in this module that we can/should test independently of the rest of the imlpementation?

Reply:
In a 2nd follow up PR, I am planning on allowing users to create their own "window indexers" to be passed into `rolling(...)`. In that PR I can add tests for these existing indexers. They have been effectively smoke tested since they get hit with every rolling test.